app/renderer/src/containers/AppContainer.js

```diff
@@ -29,19 +39,42 @@ const AppContainer = class extends Component {
     }
   }
 
+  updateQuery = q => {
+    this.setState({ q });
+    ipcRenderer.send(IPC_QUERY_COMMAND, { q });
+  };
+
+  resetQuery = () => {
+    // @todo - just clearn the results once results actions/reducers are removed
+    this.props.resetResults && this.props.resetResults();
```

Comment:
`resetResults` is missing from propTypes?

Reply:
It's there :)

Reply:
höh